integrations/DCMFloodlight/browser.js

```diff
@@ -103,155 +124,73 @@ class DCMFloodlight {
     countingMethod = countingMethod.trim().toLowerCase().replace(/\s+/g, "_");
 
     // find conversion event
-    // some() stops execution if at least one condition is passed and returns bool
     // knowing cat (activityTag), type (groupTag), (counter or sales), customVariable from config
-    event = event.trim().toLowerCase();
-    const conversionEventFound = this.conversionEvents.some(
-      (conversionEvent) => {
-        if (
-          conversionEvent &&
-          conversionEvent.eventName &&
-          conversionEvent.eventName.trim().toLowerCase() === event
-        ) {
-          if (
-            isNotEmpty(conversionEvent.floodlightActivityTag) &&
-            isNotEmpty(conversionEvent.floodlightGroupTag)
-          ) {
-            this.activityTag = conversionEvent.floodlightActivityTag.trim();
-            this.groupTag = conversionEvent.floodlightGroupTag.trim();
-          }
-          salesTag = conversionEvent.salesTag;
-          customFloodlightVariable = conversionEvent.customVariables || [];
-          return true;
-        }
-        return false;
-      }
+    const conversionEvent = this.conversionEvents.find(
+      (cnEvent) =>
+        cnEvent &&
+        cnEvent.eventName &&
+        cnEvent.eventName.trim().toLowerCase() === event.toLowerCase()
     );
 
-    if (!conversionEventFound) {
+    if (!conversionEvent) {
       logger.error("[DCM Floodlight]:: Conversion event not found");
       return;
     }
 
+    if (
+      conversionEvent.floodlightActivityTag &&
+      conversionEvent.floodlightGroupTag
```

Comment:
```suggestion
      isNotEmpty(conversionEvent.floodlightActivityTag) &&
      isNotEmpty(conversionEvent.floodlightGroupTag)
```
These fields can be empty

Reply:
```
if (
      conversionEvent.floodlightActivityTag &&
      conversionEvent.floodlightGroupTag
    )
```
This will check for empty strings also. We don't need to use `isNotEmpty` function.
<img width="1449" alt="Screenshot 2022-10-26 at 11 06 39 PM" src="https://user-images.githubusercontent.com/60897972/198097096-dcaa1ad4-124f-4a06-8c76-315990af3a02.png">